### DISCUSSION TITLE
How to do "git rebase --continue"

### DISCUSSION THREAD

**Discussion by Fish1** [jesseduffield/lazygit#2964] [Q&A]
I am trying to figure out how to initiate a "git rebase --continue" here. I am not really sure what shortcut needs to be pressed. Could someone help with that?

![image](https://github.com/jesseduffield/lazygit/assets/13600696/8335289b-6a6d-4eb1-965d-d0f7c29ac0cb)

**Comment by stefanhaller** [comment]
It's in the `view merge/rebase options` menu. The fastest way to do it is to press m, enter.
